api/post_test.go

```diff
@@ -138,6 +138,37 @@ func TestCreatePost(t *testing.T) {
 	}
 }
 
+func TestCreatePostWithCreateAt(t *testing.T) {
+
+	// An ordinary user cannot use CreateAt
+
+	th := Setup().InitBasic()
```

Comment:
Just to clarify, you're currently creating two `TestHelper`s along with two sets of test users, teams, etc by calling `Setup()` multiple times. That's not a huge deal, but you only really need one here, so you can change this to make it a bit simpler.
```go
// An ordinary user cannot use CreateAt
th := Setup().InitBasic().InitSystemAdmin()
Client := th.BasicClient
channel1 := th.BasicChannel

// first test goes here

// But a System Admin user can
SysClient := th2.SystemAdminClient

// second test goes here
```

If you've got any questions, feel free to post back here or message me in the Developers channel on https://pre-release.mattermost.com where I'm `@harrison`.